course/templates/course/course_page.html

```diff
@@ -178,11 +179,78 @@ <h5> Course Reviews :</h5>
         <br><br>
         <h7> Avrage Rating : {{course.avg_rating_for_course}} / 5</h7>
         <br><br>
-        <button type='submit' class="btn btn-primary btn-rounded btn-lg" disabled>
-          Add Review
-        </button>
+        {% if user != course.teacher_id %}
+            {% if review_view %}
+                    <button type="button" class="btn btn-primary btn-rounded btn-lg" onclick="toggleReviewForm()">Update Review</button>
+                <br><br>
+                <form action='{{course.course_id}}/delete_review' method="POST">
+                {% csrf_token %}
+                    <button type="submit" class="btn btn-primary btn-rounded btn-lg">Delete Review</button>
+                </form>
+            {% else %}
+                    <button type="button" class="btn btn-primary btn-rounded btn-lg" onclick="toggleReviewForm()">Add Review</button>
+            {% endif %}
+        {% endif %}
+      </div>
 
+    {% if  messages%}
+        <div id="reviewFormContainer" class="col-5">
+    {% else %}
+        <div id="reviewFormContainer" class="col-5" style="display: none;">
+    {% endif %}
+        <div class="col-5">
+        {% if review_view %}
+            <form action="{{course.course_id}}/update_review" method="post">
+        {% else %}
+            <form action="{{course.course_id}}/add_review" method="post">
+        {% endif %}
+                {% csrf_token %}
+                <div class="form-group">
+                    <label for="id_rating">Rating:</label>
+                    <div class="star-rating">
+                    {% if form.rating.value %}
+                        {% with form.rating.value|add:'0' as rating_value %}
+                        {% for i in "54321" %}
```

Comment:
what does thin number represent?

Reply:
Interesting way to display it. maybe change it to a simple list, just for readability.

Reply:
It appears that not possible to create a inline list inside the template itself.
https://stackoverflow.com/questions/4395230/building-a-list-in-django-templates.
There is another way to change it ?

Reply:
Would you prefer if I pass it as a list from the view function to the html file?

Reply:
I think the way you implemented is the supported way to do `for range` without implementing a custom range function.
@Omeramsc WDYT? I think we can keep it as it is

Reply:
Yes, we can, nothing wrong with the current implementation. :)